### ISSUE TITLE
Automatic mixed precision works worse for 3D neural networks.

### ISSUE THREAD

**Issue by donnyyou** [pytorch/pytorch#63313]
## 🐛 Bug

When we train 3D neural networks, the automatic mixed precision training always works worse (>1%) than full precision training. But the same setting works for 2D neural networks.

**Comment by ngimel** [comment]
cc @ptrblck fyi, could it be that some 3d functions are missing from the full precision list?

**Comment by ptrblck** [comment]
This could be the case.
@donnyyou would it be possible to post the model definition so that we could check the used operations and their support?
I also assume that "works worse" means the final accuracy is worse using mixed-precision training compared to the `float32` run?
